redi-connect-front/src/pages/front/signup-existing/ResetPassword.tsx

```diff
@@ -20,7 +20,6 @@ import {
 import { history } from '../../../services/history/history'
```

Comment:
Please remove the entire containing directory, and routes pointing to it. It's deprecated and can safely be removed.

Reply:
You have already mentioned this and there is a task for it. 
Will be done within one of the next PRs.

Reply:
Good, forgot this. Then all good - thanks!